app/src/main/AndroidManifest.xml

```diff
@@ -117,6 +117,24 @@
                     android:host="pcast.pocketcasts.net"
                     android:scheme="https"/>
             </intent-filter>
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW"/>
+                <category android:name="android.intent.category.DEFAULT"/>
+                <category android:name="android.intent.category.BROWSABLE"/>
+                <data
+                    android:host="pocket-casts-main-development.mystagingwebsite.com"
+                    android:scheme="https"
+                    android:pathPrefix="/redeem-guest-pass" />
+            </intent-filter>
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW"/>
+                <category android:name="android.intent.category.DEFAULT"/>
+                <category android:name="android.intent.category.BROWSABLE"/>
+                <data
+                    android:host="pocketcasts.com"
+                    android:scheme="https"
+                    android:pathPrefix="/redeem-guest-pass" />
```

Comment:
Should we make this future-proof and allow query parameters? See: https://github.com/Automattic/pocket-casts-android/pull/2559